### DISCUSSION TITLE
Clarify "Patched versions" for GHSA-3hxg-fxwm-8gf7

### DISCUSSION THREAD

**Discussion by abscondment** [reactiveui/refit#1932] [General]
Looking at CVE-2024-51501:

* [https://nvd.nist.gov/vuln/detail/CVE-2024-51501](https://nvd.nist.gov/vuln/detail/CVE-2024-51501) says `7.2.22` and `8.0.0` contain a fix.
* https://github.com/reactiveui/refit/releases/tag/7.2.22 indicates that it contains a fix
* [this version of https://github.com/reactiveui/refit/security/advisories/GHSA-3hxg-fxwm-8gf7](https://github.com/reactiveui/refit/security/advisories/GHSA-3hxg-fxwm-8gf7) indicates that **only `8.0.0`** contains a fix.
  * GHSA-3hxg-fxwm-8gf7 correctly starts the fix at `7.2.22`

Can we update all versions of the Github Advisory to match?

**Comment by glennawatson** [comment]
Confirming the fix landed in both 7.2.22 and 8.0.0. The CVE record and GHSA-3hxg-fxwm-8gf7 are correct in listing 7.2.22 as the start of the patched 7.x line; if any mirrored copy of the advisory still shows only 8.0.0 that is stale advisory metadata, not a packaging difference. If you are on 7.x, upgrade to 7.2.22 or later. Closing as this is advisory metadata rather than a code change.
